### ISSUE TITLE
FE: stop sharing screen (browser button)

### ISSUE THREAD

**Issue by clararfe** [montevideo-tech/hybrid-meetup-platform#107]
Currently the user can stop share screen only by clicking on the corresponding button on the webapp (the one that is between mic and leave room).

It is necessary to let the user to also stop sharing by clicking on the button that is provided by the browser.
For example, on chrome:

![Image](https://user-images.githubusercontent.com/98163437/221012249-156245ca-4b63-4565-908b-511420c57205.png)

**Comment by mpollingerQualabs** [comment]
Apparently it could be solved by listening to the `ended` or `inactive` event, like so:
`.addEventListener('ended', () => { }`

**Comment by sebastianpiq** [comment]
I think this is solved in #102 

frontend/src/views/Room.jsx line 214
```
stream.getVideoTracks()[0]
        .addEventListener('ended', () => {
          newScreenRoom.leave();
        });
```

**Comment by clararfe** [comment]
Yes, it was implemented on #102. I'll close this issue.
